app/modules/annotations/schemas.py

```diff
@@ -56,3 +56,80 @@ class Meta(BaseAnnotationSchema.Meta):
             Annotation.updated.key,
             'asset_src',
         )
+
```

Comment:
Why are these methods here and not on the annot model itself?
Note I'm thinking of moving the cache to the different classes so that these caches can be reused

Reply:
simply put, i did not want to make these methods which access properties _from the edm_ publicly usable (yet?).  my guess is they would tend to be expensive to use -- even with my vague attempt at caching edm values.   if/when something needs to use them, i would be fine with (cautiously) moving them to the actual Annotation model -- but perhaps with a bit more consideration of cost to get the values.   (and once edm goes away, this would be a moot point and i would be all for having them on the model.)

so since -- for now -- the only place that needs to get this is for this schema; i just kind of plunked them here "for internal use only" sorta thing.  (knowing the cost was worth bearing for when it needed to be indexed -- which hopefully will ultimately be done as minimal as possible.)

Reply:
I see the reasoning (for the first two methods but not the latter six, unless I'm mistaken they don't use EDM)
Regarding the implementation, I suspect I will need to do something equivalent for individual, Encounter and Sighting so I think get_locationId_from_edm on the encounter model may be better.

Reply:
addressing in refactored code